pegasus/helpers/section_api_helpers.rb

```diff
@@ -266,6 +292,10 @@ def self.valid_script_id?(script_id)
     valid_scripts.find {|script| script[:id] == script_id.to_i}
   end
 
+  def self.valid_course_id?(course_id)
+    valid_courses.find {|course| course[:id] == course_id.to_i}
```

Comment:
I'm emulating the behavior of valid_script_id above. I suspect this is because we get course/script id's as params, which are strings. (Will also add param tags to both methods).